### PR TITLE
[Translations] [Indonesian]: config

### PR DESCRIPTION
**Pull request by CikiMomogi**
Translates:
- [x] The project title and description
- [x] The nav bar
- [x] The search bar
- [ ] Fix the anchor links

### CODE REVIEW

**Comment by KnorpelSenf** [comment]
Can you open pull requests against the `main` branch of your fork? That way, all translations will be kept in your fork until the website is ready. We will still follow the progress in your fork. When everything is ready, we can do one big pull request from the fork against this repo.

File: site/docs/.vuepress/config.ts [hunk 1]
```diff
@@ -712,6 +718,346 @@ export default defineUserConfig({
...
+                    link: "/id/plugins/console-time.md",
+                  },
+                  {
+                    text: "Middleware Bermanfaat",
+                    link: "/id/plugins/middlewares.md",
+                  },
+                  {
+                    text: "[Kirimkan PR Anda!]",
+                    link:
+                      "/id/plugins/README.md#submitting-your-own-package-to-the-docs",
```
Comment (KnorpelSenf) [inline site/docs/.vuepress/config.ts:945]: The anchor needs to be adjusted to the headline in the respective article.

**Comment by CikiMomogi** [comment]
> Can you open pull requests against the `main` branch of your fork? That way, all translations will be kept in your fork until the website is ready. We will still follow the progress in your fork. When everything is ready, we can do one big pull request from the fork against this repo.

Okay.

**Comment by KnorpelSenf** [comment]
Closing in favour of https://github.com/CikiMomogi/website/pull/6

**Pull request closed without merging**